apps/src/templates/certificates/Certificate.jsx

```diff
@@ -1,4 +1,3 @@
-/* global dashboard */
```

Comment:
I see this in our codebase, but can't figure out what it's doing. Can you provide context here?

Reply:
without it, `dashboard.foo` and `window.dashboard.foo`, etc would be lint errors, because global variables on the `window` object are an antipattern. I can remove it now because I'm removing `dashboard.CODE_ORG_URL` below.